whiteblock/cmd/transactions.go

```diff
@@ -86,6 +105,33 @@ Optional Parameters:
 	},
 }
 
+var sendToTxCmd = &cobra.Command{
+	// Hidden: true,
+	Use: "to",
+	Short: "Send transaction data to an account",
+	Long: `
+The user must specify the flags that will be used for sending transaction data.
+Send a transaction data to an account.
+
+Required Parameters: 
+	--destination <address> --value <amount> --data <transaction data>
+	`,
+	Run: func(cmd *cobra.Command, args []string) {
+		command := ""
```

Comment:
You can just put `command := "send_to", our software is able to figure out the rest on the backend